src/watchNPM.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request-promise')
+const _ = require('lodash')
+const ChangesStream = require('changes-stream')
+const clean = require('normalize-registry-metadata')
+const logger = require('~/src/logging').logger(module)
+
+// will hit up registry
+exports.checkWatcher = async function () { // TODO: fill the skeleton
+  console.log('todo')
+}
+
+// will publish to mq
+exports.handleChanges = async function ({channel, client, registry}) {
+  logger.info('saw change....todo handle logic')
+}
+
+exports.startNPMWatcher = async function ({channel, client, registry}) {
+  console.log(registry)
+  client.get('changes-stream', function (err, reply) {
+    if (err) {
+      logger.error('error when hitting redis')
+      throw Error(err)
+    }
+    logger.info('Reply from Redis: ' + JSON.stringify(reply))
+    logger.info('starting changes stream')
+
+    const changes = new ChangesStream({
+      db: registry,
+      since: 'now',
```

Comment:
Just making a note that we may want to store information about the last time we got data from the stream. We can use that to pick up at a reasonable and ensure that we don't end up missing any changes. We will do this in a future PR.